src/consensus/consensus.h

```diff
@@ -6,11 +6,38 @@
 #ifndef BITCOIN_CONSENSUS_CONSENSUS_H
 #define BITCOIN_CONSENSUS_CONSENSUS_H
 
+#include "consensus/params.h"
+
+#include <stdint.h>
+
+class CTransaction;
+class CValidationState;
+
+/**
+ * Consensus validations:
+ * Check_ means checking everything possible with the data provided.
+ * Verify_ means all data provided was enough for this level and its "consensus-verified".
+ */
+namespace Consensus {
```

Comment:
Because we don't indent namespaces: https://github.com/bitcoin/bitcoin/blob/master/src/.clang-format#L31